runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -197,6 +197,7 @@ ItemChargeConfig getConfig(ConfigManager configManager)
 	protected void startUp()
 	{
 		overlayManager.add(overlay);
+		this.clientThread.invokeLater(this::updateExplorerRingChargesFromVarbit);
```

Comment:
We typically don't use `this.` for referencing fields unless they are shadowed, so this can call `clientThread.invokeLater();` without the extra specifier.

Also, since this code is only called here, it can be an anonymous function instead, ex.

```java
clientThread.invokeLater(() ->
{
	updateExplorerRingCharges(client.getVarbitValue(Varbits.EXPLORER_RING_ALCHS));
});

Reply:
Very true, I'll fix that.

Reply:
Fixed now.